public/game/styles/game.styl

```diff
@@ -15,22 +15,24 @@
 		display flex
 		justify-content space-between
 		align-items stretch
-		flex-flow row nowrap
-
-	.controls-selector
-		flex-grow 5
+		flex-wrap wrap
+		margin .8em 0
 
 	.timespan
 		font-size 3em
-		margin-left 0.5em
+		margin-left 0.3em
```

Comment:
We don't have any guidelines on leading zeroes, but let's keep it consistent :cat: